src/preset-themes/jsonSchema/jsonSchema.js

```diff
@@ -0,0 +1,227 @@
+import { makeLog } from "../../lib/utils";
+
+const log = makeLog("background");
+
+var Ajv = require("ajv");
+var ajv = new Ajv({
+    allErrors: true
+});
+
+var schema = {
+    "properties": {
+        "title": {
+            "type": "string"
+        },
+        "colors": {
```

Comment:
I didn't see this below in the schema, but we should mark "title" and "colors" as required properties.

Reply:
okay